src/parser/parse.mly

```diff
@@ -1159,7 +1159,7 @@ range:
 /*                       Miscellanous, tools                                   */
 /******************************************************************************/
 
-%inline string:
+string:
```

Comment:
Skimming http://gallium.inria.fr/~fpottier/menhir/manual.html#sec%3Ainline

Not sure why this was marked `%inline` in the first place. 
And now sure why it should no longer be marked as such.

Can you say a bit about this change?

Reply:
I don't think it makes any difference (it's not even used for the `Const_string` case), and I was trying to remove the uses of inline because the issue I found in #1955, but perhaps that deserves a separate discussion.